extensions/Worker.Extensions.Abstractions/DefaultValueAttribute.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Annotations
```

Comment:
I think dotnet doesn't use `Annotations` word. Maybe attributes? Or honestly, even if this is at `Microsoft.Azure.Functions.Worker.Extensions.Abstractions`, that's probably better than `Microsoft.Azure.Functions.Worker`, as users wouldn't ever have to use `...Extensions.Abstractions` namespace.

Reply:
Good call - i also realized that the namespace `Microsoft.Azure.Functions.Worker` was a complete copy paste accident!